build_multi_arch_docker.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# This script builds and deploys multi-architecture docker images.
+#
+# Can be run stand-alone from the command line like so:
+#   DOCKER_USER=... DOCKER_PASSWORD=... DOCKER_BASE=... DOCKER_PLATFORMS=... \
+#   TAGS=... \
+#   (source ./build_multi_arch_docker.sh; build-multi-arch-docker::main)
+# Where the environment variables are:
+#   DOCKER_USER: Docker Hub user name to which to push images.
+#   DOCKER_PASSWORD: Docker Hub password for above user.
+#   DOCKER_BASE: Docker image base name (user/image).
+#   TAGS: A set of image tags to which to deploy. Appended to DOCKER_BASE.
+#   DOCKER_PLATFORMS: Docker buildx platforms (see `docker buildx ls`).
+# Example:
+#   DOCKER_USER=foo DOCKER_PASSWORD=bar DOCKER_BASE=foo/shfmt TAGS=latest \
+#   (source ./build_multi_arch_docker.sh; build-multi-arch-docker::main)
```

Comment:
Is this source+namespacing method some form of standard? It seems kinda awkward and complex to me.

`./script action` seems much simpler and cleaner to me than `(source script && script::action)`.

Reply:
build_multi_arch_docker.sh is now folded into test.yml